docs/api_extra.rst

```diff
@@ -824,6 +881,17 @@ section <ndarrays>`.
       ``shape()``, ``stride()``, and ``operator()`` following the conventions
       of the `ndarray` type.
 
+   .. cpp:function:: auto cast(rv_policy policy = rv_policy::automatic_reference, handle parent = {})
+
+      The expression ``array.cast(policy, handle)`` is almost equivalent to
+      :cpp:func:`nb::cast(array, policy, handle) <cast>`.
+
+      The main difference is that the return type of :cpp:func:`nb::cast
+      <cast>` is :cpp:class:`nb::object <object>`, which renders as a rather
+      non-descriptive ``object`` in Python bindings. The ``.cast()`` method
+      instead returns a wrapper type, whose signature matches that of the
+      original nd-array.
```

Comment:
```suggestion
      instead returns a wrapper type that will be rendered in function signatures
      with the type of the original nd-array.
```
I didn't understand at first what it meant for the wrapper type to have a signature